src/docker/eventListener.test.ts

```diff
@@ -473,6 +474,8 @@ describe("docker/eventListener.ts EventListener methods", function () {
     // stub the setContextValue and localKafkaConnected.fire methods so we can assert that they're called
     const setContextValueStub = sandbox.stub(contextValues, "setContextValue").resolves();
     const localKafkaConnectedFireStub = sandbox.stub(localKafkaConnected, "fire");
+    // stub updateLocalConnection since we don't care about the actual connection update for this test
+    sandbox.stub(localConnections, "updateLocalConnection").resolves();
```

Comment:
This also broke in the update to VS Code 1.97.0, but I'm not entirely sure why it wasn't causing `FetchError`s before, since this should've been stubbed originally. 🤷